cloud/libvirt/actuators/machine/utils/domain.go

```diff
@@ -227,6 +231,73 @@ func setCoreOSIgnition(domainDef *libvirtxml.Domain, ignKey string) error {
 	return nil
 }
 
+const maxFWCFGChunkSize = 1024
```

Comment:
`...FW...` is firewall? firmware? Wasn't obvious to me.

Reply:
firmware

Reply:
In which case `Firmware` in the name would help.